setup/gcp/installer/forseti_installer.py

```diff
@@ -172,23 +184,19 @@ def wait_until_vm_initialized(self, vm_name):
             vm_name (str): Name of the VM instance.
         """
 
-        installation_type = self.config.installation_type.capitalize()
-        utils.print_banner('{} VM Initialization'.format(installation_type))
+        installation_type = self.config.installation_type
+        utils.print_banner('{} VM Initialization'.format(
+            installation_type.capitalize()))
         _, zone, name = gcloud.get_vm_instance_info(vm_name)
 
-        # VT100 control codes, use to remove the last line
-        erase_line = '\x1b[2K'
+        status_tracker = lambda: gcloud.check_vm_init_status(name, zone)
```

Comment:
Why do we need a lamda function to call `gcloud.check_vm_init_status(name, zone)` here?